File: src/main/no/nav/pto/veilarbfilter/client/dto/PortefoljeEnhet.java
package no.nav.pto.veilarbfilter.client.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Accessors(chain = true)
public class PortefoljeEnhet {
    String enhetId;
    String navn;
}

File: src/main/no/nav/pto/veilarbfilter/client/dto/PortefoljeEnhet.kt
package no.nav.pto.veilarbfilter.client.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import no.nav.common.types.identer.EnhetId

@JsonIgnoreProperties(ignoreUnknown = true)
class PortefoljeEnhet(
    var enhetId: EnhetId,
    var navn: String
)